designer/client/components/Fee/FeeItems.jsx

```diff
@@ -131,9 +147,32 @@ export class FeeItems extends React.Component {
       });
   };
 
+  itemHasStaticQuantity = (item) => {
+    console.log("item multiplier: ", item.multiplier);
+    return !!parseInt(item.multiplier);
+  };
+
+  changeQuantityType = (event, item, index) => {
+    event.preventDefault();
+    const { fields } = this.props;
+    let itemsCopy = [...this.state.items];
+    let itemCopy = { ...item };
+    if (this.itemHasStaticQuantity(item)) {
+      itemCopy.multiplier = Object.keys(fields)[0] ?? "1";
+    } else {
+      itemCopy.multiplier = "1";
+    }
```

Comment:
This shouldn't be necessary. In the runner:

1. if there is no multiplier present, take the fee value
2. if the multiplier is present, read the field's value, and multiply the fee value 

multiplier as in "the field which will multiply the value" 

The fee is omitted if fee.condition evaluates to false.

The designer should/would not know what the user is going to multiply by